base/typography/_links_config.scss

```diff
@@ -35,5 +36,13 @@ $active: '&:active';
       line-height: 1;
       margin-left: 2px;
     }
+  } @else if $fancy-before {
+    &:before {
+      content: "‹";
+      color: inherit;
+      font-style: normal;
+      line-height: 1;
+      margin-right: 2px;
+    }
```

Comment:
Another option could be to have one param `$fancy` and have it accept either `before